lamagraph-core/src/INet/Net.hs

```diff
@@ -0,0 +1,12 @@
+module INet.Net where
+
+import Clash.Prelude
+
+data AgentSimpleLambda
+  = Apply
+  | Abs
+  | Eps
```

Comment:
`erase` in [Lafont](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://dl.acm.org/doi/pdf/10.1145/96709.96718&ved=2ahUKEwiGzeX5kuiJAxXZQFUIHQ7OKCsQFnoECA4QAQ&usg=AOvVaw27iT-kxOCWNPNI4s7HFRix)
`delete agent` in [Interaction nets in Ocaml](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://joerg.endrullis.de/downloads/gcm2024/STAF_2024_paper_72.pdf&ved=2ahUKEwjG_OKtk-iJAxXCgSoKHXcVFB4QFnoECB0QAQ&usg=AOvVaw3Y3w9zQK1XRkjotODxVhNx)